modules/post/windows/gather/screen_spy.rb

```diff
@@ -127,11 +139,11 @@ def run
 
   end
 
-  def migrate_explorer
+  def migrate
     pid = session.sys.process.getpid
     session.sys.process.get_processes.each do |p|
-      if p['name'] == 'explorer.exe' and p['pid'] != pid
-        print_status("Migrating to explorer.exe pid: #{p['pid']}")
+      if p['name'] == datastore['PROCESS'] and p['pid'] != pid
+        print_status("Migrating to #{datastore['PROCESS']} pid: #{p['pid']}")
```

Comment:
My one question here is what happens if we have multiple processes with the same name? Say I provide a name such as `Chrome.exe`, which often has multiple instances of itself running at any given time. Would this not just migrate to the first instance of one of those processes vs a specific one? That would be my only concern here.

Reply:
The way the logic is currently setup it will definitely default to the first running instance it runs into. 

Off the top of my head, removing process name and requiring a PID could be substituted instead of process name.

Reply:
Yeah that may be a better option just given that otherwise one may end up migrating into a session with different restrictions then they expect. Just a thought though, given that some processes may have the same name but different restrictions applied to each process. I would expect this to be less common of an issue but still something that may come up from time to time so thought it was worth at least discussing.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/14994/commits/0b06904dd020256ef43c039bf4d03dd01ef251cb